cypress/e2e/iframe.cy.ts

```diff
@@ -3,14 +3,13 @@ import {IframePage} from "../page/index";
 describe("Working with Iframe", () => {
   let iframePage: IframePage;
   const htmlText = "HTML Tutorial";
-  const cssText = "CSS";
   beforeEach(() => {
     iframePage = new IframePage();
   });
 
   it("Iframe page", () => {
     iframePage.visit();
     iframePage.getFrameTitle(htmlText);
-    iframePage.goToCssPageInFrame(cssText);
+    iframePage.goToCssPageInFrame();
   });
```

Comment:
we should have two tests (two `it`s):

1. to open the HTML Tutorial, and verify that frame title
2. to open the CSS Tutorial and verify that frame title